.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: ruby
 rvm:
-  - 2.3.3
+  - 2.1.2
```

Comment:
Copy what we have in Portus. We are testing more versions there. And don't remove ruby-head